### ISSUE TITLE
Swipe mode always multiple

### ISSUE THREAD

**Issue by chowaikong** [Malinskiy/SuperRecyclerView#85]
Here is the bug, not only I set the swipe mode to single but also not setting any mode, these item can still swipe multiple, what can I do or what I do wrong? My code is similar to your demo.

**Comment by Null45** [comment]
Confirm. Also swipe mode not saved on orientation change and adapter.getOpenItems() always returns -1 for all items. adapter.OpenItem(0) doesnt works too.

**Comment by enginebai** [comment]
Yes, I encountered the same problem, `BaseSwipeAdapter.setMode(Mode.Single);` is useless at all.
